tests/unit/test_main.py

```diff
@@ -86,7 +86,7 @@ def test_create_flow_mod_10(self, *args):
     @patch('napps.kytos.of_l2ls.main.FlowMod13')
     def test_create_flow_mod_13(self, *args):
         """Test _create_flow_mod method for flow_mod 1.3 packet."""
-        (mock_flow_mod, mock_oxm_tlv, mock_action, mock_instruction) = args
+        (mock_flow_mod, mock_oxm_tlv, _, mock_instruction) = args
```

Comment:
It seems that this `mock_action` variable corresponds to the patched `Output13` above it. Can you test removing the line below (_and_ the `mock_action` parameter)?

`    @patch('napps.kytos.of_l2ls.main.Output13')`